minecode/tests/test_debian.py

```diff
@@ -332,20 +344,20 @@ def test_parse_packages_from_installed_status(self):
 
 class DebianLSLRTest(BaseDebianTest):
 
-    @expectedFailure
     def test_DebianDirectoryIndexVisitor_from_debian(self):
         uri = 'http://ftp.debian.org/debian/ls-lR.gz'
-        test_loc = self.get_test_loc('debian/lslr/ls-lR_debian.gz')
+        temp_gz_location = self.get_tmp_gz_file('debian/lslr/ls-lR_debian')
```

Comment:
This test still fails: https://github.com/nexB/purldb/actions/runs/4310196055/jobs/7518392136#step:6:764

You need to do something like this:
```suggestion
        test_loc = self.get_test_loc('debian/lslr/ls-lR_debian')
        temp_gz_location = self.get_tmp_gz_file(test_loc)
        with patch('requests.get') as mock_http_get:
            mock_http_get.return_value = mocked_requests_get(uri, temp_gz_location)
            uris, _, _ = debian_visitor.DebianDirectoryIndexVisitor(uri)
```

Reply:
Thanks @JonoYang, I've updated the tests, and the original lsLr files are used, and the json result have been updated accordingly.